agent/command/shell.go

```diff
@@ -117,10 +117,14 @@ func (c *shellExec) Execute(ctx context.Context, _ client.Communicator, logger c
 	logger.Execution().Debug("Preparing script...")
 
 	var err error
+	var originalScript string
 	if err = c.doExpansions(conf.Expansions); err != nil {
 		logger.Execution().Warning(err.Error())
 		return errors.WithStack(err)
 	}
+	if originalScript != c.Script {
+		logger.Task().Warning("`${SHELL_EXPANSION}` syntax will not work due to evergreen expansions. See Evergreen FAQ for details.")
```

Comment:
maybe "will only work for Evergreen expansions" since that could be what users are expecting?

Reply:
This doesn't look updated yet to me

Reply:
This doesn't look updated to me